File: CS/Form1.cs
using System;
using System.Data;
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace PieChart {
    public partial class Form1 : Form {
        public Form1 () {
            InitializeComponent();
        }

        private void Form1_Load (object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'gspDataSet.GSP' table. 
            // You can move, or remove it, as needed.
            this.gSPTableAdapter.Fill(this.gspDataSet.GSP);

        }

        private void chartControl1_CustomDrawSeriesPoint(object sender, CustomDrawSeriesPointEventArgs e) {
            e.LegendText += " (" + ((DataRowView)e.SeriesPoint.Tag)["GSP"].ToString() + ")";
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Data
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
Imports DevExpress.XtraCharts.Designer.Native

Namespace PieChart
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
			' TODO: This line of code loads data into the 'gspDataSet.GSP' table. 
			' You can move, or remove it, as needed.
			Me.gSPTableAdapter.Fill(Me.gspDataSet.GSP)

		End Sub


	End Class
End Namespace
